### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 20th July 15:00UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#1039]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - [FCMP++ update](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/449#note_25409)
    - [request for multisig solution + candidates](https://libera.monerologs.net/monero-community/20240717#c399026) - luigi1111
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [Monero Moon](https://www.themoneromoon.com/)    
5. [CCS updates](https://ccs.getmonero.org/)    
  a. [ofrnxmr feat. BasicSwapDEX - take over the world pt 2](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457)    
  b. [Monero Poker Infrastructure Project](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/461)    
  d. [rehrar help out the project](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/470)    
  e. [VOSTOEMISIO - FCMP Animated Explainer Video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/472)    
  f. [VOSTOEMISIO - Remake Introductory "What Is Monero?" Video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/473)    
  h. [monerobull for website workgroup](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/476)    
  i. [From Prototype to Marketplace: Maturing the XMR-BTC Atomic Swaps Ecosystem](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477)    
  k. [yamabiiko full-time work on Cuprate (2 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/479)    
6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2025](monerokon.org)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time
    - Rucknium requests feedback on this [MAGIC funding proposal](https://github.com/MAGICGrants/Monero-Fund/issues/32) from from Paga    
9. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/1032)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Logs 
> __< plowsof >__ Luigi didnt say he got hacked and has cooperated with handing over images (even though IMO that avenue is a dead end)      

> __< geonic >__ Tx plow     

> __< geonic >__ It’s a dead end because it wasn’t a hack     

> __< plowsof >__ geonic no update on the FFS wallet (or that particular overfunding) i think a meta issue with all relavant info in one place would help      

> __< geonic >__ Ok I’ll have to make one then     

> __< plowsof >__ If it was an inside job : the perp is handing the evidence he had infinite time to prepare      

> __< 3​21bob321:monero.social >__ the image if it was taken at the time of the "incident" would prove or disprove a hack     

> __< plowsof >__ Nope , wont prove anything     

> __< geonic >__ Agreed     

> __< d​ave.jp:matrix.org >__ Don’t worry we will find where funds went after quantum computers break it     

> __< geonic >__ Can someone set up a prediction market where we can bet on who stole it between Luigi and fluffy?     

> __< plowsof >__ Removing opinions of what haopened (because we cant prove anything) we have moved to a new setup removing all plausible deniability      

> __< plowsof >__ And await multisig      

> __< geonic >__ Meaning currently only luigi has keys     

> __< geonic >__ that says something doesn’t it     

> __< geonic >__ Regarding project coral reef… Dan Teree is a cofounder of Tari Labs and a veteran executive at Ticketmaster. Later founded another ticketing platform, Ticketfly. Obviously a guy with deep connections in the music space.     

> __< geonic >__ I’d be willing to bet that the integration of Monero in the dozen or so musician stores was done either at no cost or very low cost, as a favor.     

> __< geonic >__ So the $500K was certainly not spent as advertised. Of course fluffy can provide receipts like I did for my CCS and prove me wrong.     

> __< geonic >__ I’ll pay him 10 xmr for the trouble     

> __< geonic >__ fluffypony what do you say? Do you accept the bet?     

> __< 3​21bob321:monero.social >__ How do you think you investigate an intrusion     

> __< 3​21bob321:monero.social >__ Hopes and dreams?     

> __< 3​21bob321:monero.social >__ Atm its broken trust between community and luigi     

> __< geonic >__ No it’s not     

> __< 3​21bob321:monero.social >__ He was holding the back     

> __< 3​21bob321:monero.social >__ Prove what happened     

> __< geonic >__ Holding back what?     

> __< 3​21bob321:monero.social >__ Bag*     

> __< plowsof >__ 2+unknown where holding the bag     

> __< 3​21bob321:monero.social >__ Currently your speculating fluddy stole it     

> __< 3​21bob321:monero.social >__ Fluffy*     

> __< geonic >__ Never heard of the unknown. There were more than 2?     

> __< 3​21bob321:monero.social >__ Currently i am speculating luigi lost it     

> __< plowsof >__ Officially it was 2 the + unknown is to account for the evil hacker     

> __< 3​21bob321:monero.social >__ How do we prove whos right     

> __< 3​21bob321:monero.social >__ Process of elimination     

> __< geonic >__ The hacker or the quantum computer     

> __< 3​21bob321:monero.social >__ Audit wallet pc     

> __< plowsof >__ We cant prove who is right , just remove plausible deniability and move on https://github.com/monero-project/meta/issues/935     

> __< 3​21bob321:monero.social >__ Duh     

> __< 3​21bob321:monero.social >__ Ok     

> __< 3​21bob321:monero.social >__ Remove luigi too. Zero trust bruh     

> __< geonic >__ Guess I forgot to vote there     

> __< geonic >__ Keeping luigi as the ccs escrow is a vote of convidence by the community and more or less exonerates him as a suspect     

> __< geonic >__ Let’s do an anonymous vote on whether fluffy stole it     

> __< plowsof >__ Scenario: there is an audit that proves luigi was not hacked - response from 'luigi did it side' would be lol proves nothing he had enough time to set this up ahead of time!!!     

> __< plowsof >__ Your honour, here is the evidence that we got sent over megaupload by the potential suspect and we forensically analy- dismissed      

> __< geonic >__ Let’s vote: will fluffy accept my offer and make $1600 for maybe an hour of work and in the process clear any suspicions about Project Coral Reef?     

> __< geonic >__ Who would pass that up     

> __< d​ave.jp:matrix.org >__ If you can get the coin to pump 100% the funds are justified     

> __< d​ave.jp:matrix.org >__ They needed to pay celebs to accept xmr and all the pr     

> __< geonic >__ It’s no less legitimate than paying for an Oscar campaign, that’s for sure     

> __< geonic >__ The question is whether the money actually went to these celebs and publicists or if it ended up elsewhere     

> __< geonic >__ Like, to use your example, Tari developers     

> __< d​ave.jp:matrix.org >__ Oscar didn’t pump the coin     

> __< d​ave.jp:matrix.org >__ Money was spent to pr xmr not a movie     

> __< d​ave.jp:matrix.org >__ I would gladly donate to coral reef 2.0     

> __< geonic >__ <d​ave.jp:matrix.org> Money was spent to pr xmr not a movie <== receipts were never shown so it’s unclear what the money was spent on     

> __< plowsof >__ Community meeting in hr50mins https://github.com/monero-project/meta/issues/1039     

> __< luigi1111 >__ <geonic> plowsof: any word from core about the 867 xmr missing from the FFS? <= what?     

> __< s​yntheticbird:monero.social >__ enter panic mode     

> __< plowsof >__ this one https://forum.getmonero.org/9/work-in-progress/2564/0mq-by-tewinget , an overfunding that was meant to be returned to the GF      

> __< plowsof >__ im going to pass you onto the bF / GF department , thank you for holding sir      

> __< luigi1111 >__ Oh. Neither bf nor I have the ffs wallet     

> __< plowsof >__ goenic mentioned there being multiple as sub addresses didnt exist? does that sound accurate (im not sure) cc fluffypony      

> __< plowsof >__ geonic*      

> __< plowsof >__ Meeting time https://github.com/monero-project/meta/issues/1039 sorry im 1 minute late      

> __< plowsof >__ luigi are the rumours true that your yachts navigation system is using windows 3.1 and unaffected by the recent global tech issues     

> __< plowsof >__ lets discuss some community highlights. waiting on a multisig solution with great UX was mentioned earlier this week https://libera.monerologs.net/monero-community/20240717#c399026 , so far jwinterm and gingeropolous have put their names in the hat      

> __< msvb-lab >__ Hello, I guess.     

> __< p​lowsof:matrix.org >__ News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [Monero Moon](https://www.themoneromoon.com/)     

> __< plowsof >__ greetings      

> __< b​oog900:monero.social >__ hi     

> __< y​amabiiko:unitoo.it >__ Hi     

> __< nioCat >__ meow     

> __< plowsof >__ hello there     

> __< plowsof >__ i notice vthor has been pushing xmrsigner along, can we follow the latest progress somewhere?     

> __< plowsof >__ i think we can move onto the ccs merge list      

> __< plowsof >__   a. [ofrnxmr feat. BasicSwapDEX - take over the world pt 2](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457)         

> __< plowsof >__ luigi left a comment 5 days ago here https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457#note_25391     

> __< plowsof >__ ofrnxmr made some changes following that      

> __< vthor >__ plowsof: not finished yet, took almost 22h to build modified monero-wallet-rpc on pi 0, and now I will (after static build runs through) finish last things UR related on the XmrSigner, close all my refactoring construction places, update the buidroot (monerosigner-os) and then I finaaly finished milestone 2     

> __< p​lowsof:matrix.org >__ 1860 xmr us only 306k usd now luigi     

> __< p​lowsof:matrix.org >__ s/us/is     

> __< plowsof >__ thanks for the update vthor, nice to see that progrss is being made finally. thank you      

> __< plowsof >__ moving on for now      

> __< plowsof >__   b. [Monero Poker Infrastructure Project](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/461)         

> __< vthor >__ after that it get's easier how most things already done for monero-gui, and milestone 3 is only cleanup, tools, refactoring and documentation.     

> __< plowsof >__ seems stale now and the "xmr.poker exists but isnt active" point is convincing enough to close this imo      

> __< plowsof >__ i notice boog900 and yamabiiko are here. i would like to jump to yamabikos proposal next      

> __< plowsof >__   k. [yamabiiko full-time work on Cuprate (2 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/479)         

> __< plowsof >__ i see that this is a cuprate work proposal but part of it will be for contributing to core docs , on ZMQ      

> __< plowsof >__ for example this old issue requesting ZMQ docs on site https://github.com/monero-project/monero-site/issues/1883      

> __< r​ucknium:monero.social >__ I said this already in #cuprate:monero.social  to yamabiiko: I was surprised that ZMQ was the highest-priority cuprate item now, but maybe that's because the cuprate core-functions development has gone so quickly. I am looking forward to the first alpha release of the cuprate binary     

> __< r​ucknium:monero.social >__ ZMQ docs are welcome. I tried to get moneros's ZMQ to work once, but I couldn't figure it out. Maybe docs with a few recipes like how to subscribe to txs from a Python script :)     

> __< y​amabiiko:unitoo.it >__ I didnt even know that issue exists, but me and boog noticed the lack of documentation so seems good     

> __< plowsof >__ what will ZMQ docs change? is it important?      

> __< b​oog900:monero.social >__ yamabiiko:  will need to go through monerod to know what the types are, it makes sense to document these types at the same time as porting them to Cuprate IMO     

> __< plowsof >__ our ZMQ implementation has issues? (or would that hopefully be discovered?) would be nice for vtnerd to follow this proposal also      

> __< y​amabiiko:unitoo.it >__ There is little to none info on how to setup ZMQ and what fields are sent for example     

> __< plowsof >__ or is this for, in the case of cryptogrampys issues, service to be created that receive near-real-time updates from monerod without having to poll for data      

> __< r​ucknium:monero.social >__ IIRC monerod's ZMQ README warns that sometimes txs might not be announced to ZMQ. Anyone know why that could happen? Can cuprate guarantee that txs get sent by ZMQ?     

> __< plowsof >__ the previous cuprate meeting https://github.com/monero-project/meta/issues/1036     

> __< plowsof >__ 3 months not 2* i forgot to update the issue sorry     

> __< b​oog900:monero.social >__ I think that's to do with the underlying ZMQ lib, so no     

> __< geonic >__ hello     

> __< p​lowsof:matrix.org >__ hello     

> __< geonic >__ luigi1111: do you remember who did have access to the FFS wallet? And whether it was one wallet or separate wallets for each proposal     

> __< plowsof >__ i think we can move on from yamabiikos proposal now? during the super official community meeting everyone is engrossed in currently geonic !!!!     

> __< geonic >__ :D     

> __< plowsof >__ xD     

> __< geonic >__ Is there a quorum? I see 3 ppl :p     

> __< vthor >__ What means quorum?     

> __< geonic >__ Enough people to hold a meeting     

> __< l​m:matrix.baermail.fr >__ 👋     

> __< plowsof >__ im sure there are other people not here keeping fingers on the pulse elsewhere and serving just as an important role to the project if they where in our presence currently      

> __< plowsof >__ ok thanks for attending yamabiiko, lets move on      

> __< plowsof >__   d. [rehrar help out the project](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/470)         

> __< geonic >__ I have a finger on the pulse meme but I'll refrain :p     

> __< geonic >__ No comments it seems     

> __< plowsof >__ 10-10 votes for rehrar, +1 from luigi to make it a landslide victory, merging rehrar      

> __< ofrnxmr >__ 😂     

> __< plowsof >__ now then     

> __< l​m:matrix.baermail.fr >__ Was there any discussion with monerobull of potential share of the work ?     

> __< ofrnxmr >__ I see a lot of ppl working on site     

> __< geonic >__ And not enough people taking over the world     

> __< ofrnxmr >__ not the people who are proposing to do so tho.     

> __< vthor >__ :D     

> __< geonic >__ maybe move rehrar to funding required and put a banner that the proposal generated significant discussion with a link to the github     

> __< rehrar >__ it's true     

> __< geonic >__ this has been done before     

> __< ofrnxmr >__ Almost a direct quote from your own ccs comments :D lol     

> __< plowsof >__ https://ccs.getmonero.org/proposals/monero-to-the-oscars.html for reference: the top of the proposal      

> __< plowsof >__ ok so rehrars merged lets move on      

> __< geonic >__ Yay     

> __< ofrnxmr >__ i already +1 diego, but im -2 if the "luogi     

> __< ofrnxmr >__ "Luigi's assistant" is the job     

> __< plowsof >__ vostoemisios animated video proposals (2) - which i think should just move into 1 to centralise the feedback (anyone feel strongly about this?)     

> __< geonic >__ I’m not clear if these are menat to replace the videos on the website?     

> __< geonic >__ meant*     

> __< plowsof >__   e. [VOSTOEMISIO - FCMP Animated Explainer Video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/472)         

> __< plowsof >__   f. [VOSTOEMISIO - Remake Introductory "What Is Monero?" Video](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/473)         

> __< ofrnxmr >__ I think 1 makes things cleaner, but its still the same number of total milestones      

> __< ofrnxmr >__ Geonic, 473 is     

> __< plowsof >__ the "Introductory" is meant to replace it yes (i assume)     

> __< r​4v3r23:monero.social >__ will he be making decisions on CCSs?     

> __< ofrnxmr >__ No     

> __< rehrar >__ all of them     

> __< r​4v3r23:monero.social >__ good     

> __< rehrar >__ I mean no     

> __< plowsof >__ vosto has done previous work so we know what we're getting, not much risk on moving to funding, its just about gathering feedback from community at each stage      

> __< ofrnxmr >__ Vost also does firo work     

> __< rehrar >__ vosto always delivers and more than a few people have commented on how dated our stuff looks     

> __< ofrnxmr >__ +1     

> __< geonic >__ These people are not our audience though     

> __< ofrnxmr >__ He did monerokon stuff too     

> __< geonic >__ all paid     

> __< ofrnxmr >__ Rehrar, are you luigis assistant     

> __< geonic >__ I’d like for vosto to be more proactive in soliciting feedback     

> __< ofrnxmr >__ Or plowsof's      

> __< plowsof >__ as the video is on the main page, i shuld think for that specific one a white background would be warranted as it would look weird .. but the content/graphics of vostos stuff are really friendly ,, not darknet money vibes at all     

> __< rehrar >__ If he asked me to do some things I would. But the primary roles are to do things he doesn't want to do independently. This takes things off of his plate.     

> __< plowsof >__ yeah, feedback gathering is the most important thing      

> __< plowsof >__ i need to lose a bit of weight so this is a welcome addition, thank you rehrar      

> __< geonic >__ it also needs to have less text than the previous videos. They’re so crammed with information as to be incomprehensible and lose the viewer. And the actor’s voice is monotone     

> __< x​mrscott:monero.social >__ Might be worth trying to do something that is also dark mode friendly should the site ever support dark mode     

> __< rehrar >__ Only about a quarter of my work would be doing what he asks. The other stuff is independently driven.     

> __< ofrnxmr >__ the tone used describe assistant sounds like youll do what luigi asks, regardless of what community wants      

> __< geonic >__ yes, export white + dark background     

> __< plowsof >__ dark and light mode we want then      

> __< geonic >__ And another VO actor     

> __< plowsof >__ and gathering feedback, no isses with merge to funding though      

> __< plowsof >__ issues*     

> __< geonic >__ he should involve us in choosing the VO actor     

> __< plowsof >__ noted      

> __< ofrnxmr >__ Which doesn't sound very appealing. Just more of the same. I'm voting for diego to be his own person, not for diego to be luigis legs     

> __< plowsof >__   h. [monerobull for website workgroup](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/476)         

> __< geonic >__ ofrnxmr: this is luigi’s proposal so …     

> __< rehrar >__ ofrn give it a trial period. Let's see how things shake out. If after we give it a go you decide I'm not able to do enough independently, that can be voiced.     

> __< plowsof >__ hardenedsteel not a fan of the lowering the temperature stuff in the comments here 😅     

> __< ofrnxmr >__ geonic: its really my proposal.. but where luigi closed mine and told diego to open his own     

> __< ofrnxmr >__ Mb has been mia on site. I cant endorse the proposal     

> __< rehrar >__ I think you'll find things turn out well     

> __< geonic >__ what’s the delineation between rehrar and monerobull     

> __< geonic >__ Luigi seems to think they can work together     

> __< ofrnxmr >__ mb said he could do the job for cheaper, but hasnt done the job or even scoped to do the job     

> __< geonic >__ neither has rehrar afaict     

> __< ofrnxmr >__ Again, 0 presence on monero-site.      

> __< ofrnxmr >__ Rehrar has code on site     

> __< plowsof >__ i think rehrar made getmonero      

> __< geonic >__ Recently I mean     

> __< ofrnxmr >__ I agree about recently. As i said a few mins back, both rehrar and mb are absent on -site atm     

> __< ofrnxmr >__ A lot of other contributors.. and we have a maintainer     

> __< geonic >__ yes, no reason to judge one and not the other     

> __< geonic >__ who’s the maintainer currently?     

> __< plowsof >__ zoom out!     

> __< rehrar >__ luigi     

> __< ofrnxmr >__ wrong     

> __< ofrnxmr >__ Its ofrnxmr in a mask     

> __< ofrnxmr >__ (its not luigi)     

> __< plowsof >__ the plan was to keep holding meetings until monerobull didnt attend one and rehrar did     

> __< geonic >__ (:     

> __< plowsof >__ monerobull was here today actually and i spoke with him (not like he's not here) just saying lol     

> __< rehrar >__ I mean, if people want to fund mb too I don't see why not     

> __< geonic >__ Yeah. Question is who does what     

> __< ofrnxmr >__ i do see why not..     

> __< plowsof >__ yes luigis comments mention both are elligible for funding and dont cancel the other     

> __< ofrnxmr >__ What are we funding? moderator position?     

> __< geonic >__ No that’s unpaid     

> __< geonic >__ I’d like to move to make mb head matrix mod though     

> __< ofrnxmr >__ so what are we funding? Nothing     

> __< ofrnxmr >__ head of moderation involved being a able to lead other mods      

> __< ofrnxmr >__ Mb wants to be a mod. I dont think the ccs implies anything to do with leading anybody     

> __< plowsof >__ its hard to go from no presence in the elite mod circle to being their boss ,, the inclusion should be gradual      

> __< geonic >__ That was one of the "positions" we were looking to fill afaik     

> __< ofrnxmr >__ fk gradual     

> __< ofrnxmr >__ capable     

> __< geonic >__ ofrnxmr: do you like mb for unpaid matrix mod?     

> __< ofrnxmr >__ yes     

> __< geonic >__ We’ve got a second     

> __< geonic >__ all in favor? :p     

> __< rehrar >__ This really isn't complicated. luigi doesn't want to lead the mod team. He asked if I'd do it. I added it to the responsibilities.     

> __< x​mrfamily:matrix.org >__ 😂😅     

> __< ofrnxmr >__ rehrar, im on boars with you leading mod team     

> __< plowsof >__ rehrar has already links / involvement with the current mod team for years      

> __< ofrnxmr >__ Rehrar already owns matrix mod team     

> __< ofrnxmr >__ (literally)     

> __< geonic >__ Yes that’s why we should distribute responsibilities     

> __< plowsof >__ finger on the pulse, and power switch      

> __< ofrnxmr >__ We need to reelect mods      

> __< ofrnxmr >__ Since current mods arent elected at all.     

> __< ofrnxmr >__ (aside from plowsof)     

> __< rehrar >__ If someone wants to take over infrastructure work from Cypher Stack, do let core team know.     

> __< plowsof >__ dropping binarybarons latest proposal here:   i. [From Prototype to Marketplace: Maturing the XMR-BTC Atomic Swaps Ecosystem](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477)         

> __< ofrnxmr >__ No, i prefer you keep if and get paid to do it @rehrar     

> __< ofrnxmr >__ Keep it*     

> __< geonic >__ And let someone else moderate for free     

> __< plowsof >__ so we need clarification on monerobulls proposal now that rehrars has been merged and weve all accepted it      

> __< geonic >__ DeCenTRaliZatioN and all that     

> __< ofrnxmr >__ im not speaking from a position of bias, but i dont like like unstoppable has been dealt with for the last year+     

> __< geonic >__ yes but I think we can agree to make him a matrix mod for now. He has agreed to do it for free     

> __< ofrnxmr >__ geonic, we need to replace mods and sort out power hierarchy      

> __< ofrnxmr >__ I vote for #monero mod (not admin)     

> __< geonic >__ Remove all mods except plowsof, add mb     

> __< ofrnxmr >__ Mb* for #monero     

> __< x​mrfamily:matrix.org >__ .... mb isn't most professional for mod position... but idc     

> __< geonic >__ and rehrar (unpaid)     

> __< geonic >__ we’re looking for effective, not professional     

> __< ofrnxmr >__ Rucknium for mrlab and mrlounge     

> __< x​mrfamily:matrix.org >__ 100% then he should be mod     

> __< v​tnerd:monero.social >__ They cannot make this guarantee, zmq drops when the connection queue is full     

> __< geonic >__ plowsof: is this in the minutes? (:     

> __< plowsof >__ when i was a wee lad, before mod powers, i was given piecemeal permissions. first mod in community on matrix and such      

> __< plowsof >__ yes     

> __< plowsof >__ i was given mod on irc, then it was revoked because i was changing room topics "to_text_like_this"     

> __< plowsof >__ then i had to go to irc school      

> __< plowsof >__ now i just dont change topics anymore      

> __< geonic >__ Yes we need to see mb’s certificate of achievement in online moderation :p     

> __< ofrnxmr >__ mb has discord certif     

> __< geonic >__ that should qualify him to moderate any cesspool     

> __< plowsof >__ discord / reddit and monero town     

> __< plowsof >__ hi vtnerd, thanks for commenting      

> __< ofrnxmr >__ Banhammer needs to demote from admin > mod in rooms where it is an admin, and really, banhammer should be on monero.social, not matrix.org     

> __< plowsof >__ hello binarybaron      

> __< ofrnxmr >__ And monerobulls mod account should be monero.social     

> __< binarybaron >__ hey everyone, einliterflasche and I are here in case any questions arise regarding our CCS proposal     

> __< geonic >__ So we have 3 Matrix mods - plowsof, monerobull, rehrar. They can decide amongst themselves who will be head mod as long as we’re clear the mod positions are unpaid     

> __< r​ucknium:monero.social >__ AFAIK, the matrix.org federation issues have mostly been solved. Big thanks to whoever worked on that ❤️     

> __< rehrar >__ The leading of the mod position is included in my proposal and I'd be billing hours for that portion     

> __< plowsof >__ (i need to disable the bot that tells people to leave matrix dot org hm)     

> __< geonic >__ It was made clear by luigi that moderator is an unpaid position     

> __< ofrnxmr >__ its not abt federation issues, but about homeserver admin power over the account. Matrix.org sholdnt have control over our rooms     

> __< rehrar >__ leading the mod team isn't     

> __< plowsof >__ i followed through by creating a monero social account then became emotionally distressed over playing matrix encryption key roulette.... i need a separate device for my social account      

> __< geonic >__ Says who?     

> __< ofrnxmr >__ leading mod team includes running the homeserver      

> __< ofrnxmr >__ Its an administration job     

> __< rehrar >__ says who it's not?     

> __< plowsof >__ binarybaron is here btw if anyone has Q's about his atomic swap proposal      

> __< r​ucknium:monero.social >__ If moderation personnel is being adjusted, can #MRL Matrix re-enable emoji reactions for all users?     

> __< geonic >__ <luigi1111> he can sure be a mod about me but that's not a paid role     

> __< ofrnxmr >__ diego isnt meant to be a mod afaict.      

> __< ofrnxmr >__ Hes meant to be the mods admin     

> __< rehrar >__ "be a mod" being the term he used thehre     

> __< geonic >__ <lluigi1111>  although I see the issue with diego listing leading mods or whatever     

> __< plowsof >__ https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477 6 monther      

> __< ofrnxmr >__ rehrar: can you agree to to allow rucknium permission to reenable emojis in mrl     

> __< ofrnxmr >__ #emojigate has gone on for far too long     

> __< plowsof >__ lol     

> __< rehrar >__ lol why was that disabled?     

> __< ofrnxmr >__ Because some mod power tripped     

> __< x​mrscott:monero.social >__ Room has Send Reactions: Default     

> __< geonic >__ You can lead mods but it’s unpaid. I’m sure you’ll find something else to bill for.     

> __< plowsof >__ MRL is a serious room, too many children leaving crying laugh emojis when oooo123 was around and those idiots had emoji perms removed ( i also had perms removed )      

> __< r​ucknium:monero.social >__ xmrscott: Most Matrix users in #MRL are set to power:1, which disables emoji reactions.     

> __< rehrar >__ unfortunately for you, you don't decide. It's in the current proposal. I have no current plans to change it.     

> __< geonic >__ Do you decide?     

> __< plowsof >__ thanks for attending binarybaron, 8 updoots and comments already. my only issue would be the length of 6 months, your previous proposals where not so long? (and one of them you where slow to complete)     

> __< geonic >__ There goes the old Diego again     

> __< geonic >__ I retract my comment to move to funding required     

> __< rehrar >__ It was one of the things luigi asked me to take on. It's one of the things I accounted for when I made up hours and price     

> __< r​ucknium:monero.social >__ I like emoji reactions because it reduces the number of redundant messages. People can 🙏, 🚀, 👍️ a bunch without sending more messages. Of course IRC cannot see them.     

> __< geonic >__ It is obvious rehrar has no plans to be accountable to the community as his ccs purports     

> __< rehrar >__ you do not = community geonic     

> __< geonic >__ I said I retract my comment, I didn’t say I’m community     

> __< geonic >__ you keep saying “luigi asked me” as if that’s the word of God. It’s not     

> __< ofrnxmr >__ +1 to geonics last msg     

> __< ofrnxmr >__ I dont enjoy hearing what luigi asked      

> __< ofrnxmr >__ s/luigi/plowsof and you sound a lot better     

> __< rehrar >__ he asked me to take those roles from him so he won't have to do them. I will be accountable to the community on whether or not they're done well. If not, he can choose a different person for the role.     

> __< ofrnxmr >__ Luigi never has had anything to do with matrix mod     

> __< nioCat >__ r​ucknium I saw your 3 emojis  :)     

> __< nioCat >__ I guess it depends      

> __< rehrar >__ there are matrix rooms with current mods. He's in them. And he's asked to make decisions on some things. And he doesn't want to do that anymore.     

> __< ofrnxmr >__ Thats not how shit should work     

> __< luigi1111 >__ I don't think the "leading the mods" needs to be that contentious. Cultivating community culture and mod relationships is the idea more or less. "moderating" is not a paid position     

> __< binarybaron >__ plowsof Looking at the scope of the CCS this is a six month project. We believe there won't be trust issues since we have already completed two CCSs. We would really like some safety that we can really spend six months on this as committing to the project means spending less time university for a whole semester.     

> __< geonic >__ So the community picks a new mod. He doesn’t.     

> __< luigi1111 >__ I'm not modding matrix, though I'm present in the room     

> __< ofrnxmr >__ Mods shouldnt go crying to luigi for permission to do anything     

> __< ofrnxmr >__ Mods should mod their own rooms to be on topic     

> __< x​mrfamily:matrix.org >__ Being a mod for matrix is like... bitch position?     

> __< geonic >__ so remove it from the “job description”. Rehrar does not get authority over mod team with this role. He's welcome to use his soft skills as much as he wants     

> __< plowsof >__ xmrfamily wanted to mention an idea for creating a "non profit monero organization that deals with advocation for smarter msb/ regulations state,national, international levels ", would this be twisted-edwards / monero-policy related?      

> __< ofrnxmr >__ Anything else should be extreme circumstances where banhammer is used due to common sense (like uploading content that is no-good)     

> __< geonic >__ https://github.com/monero-project/meta/issues/583     

> __< x​mrfamily:matrix.org >__ Sorry about profanity     

> __< geonic >__ Rehrar is ineligible to hold a leadership position as long as this statement stands     

> __< geonic >__ pay him for services if you want     

> __< ofrnxmr >__ we should have an election then     

> __< geonic >__ nah, just hold a vote. Mb said he’ll do it for free. Let him do it.     

> __< plowsof >__ a open end to the meeting here, please continue and thank you all for attending     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
